.github/workflows/mysql.yml

```diff
@@ -0,0 +1,45 @@
+name: mysql
+
+on: [push, pull_request]
+
+jobs:
+  mysql_test:
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest]
+        ruby: [2.5, 2.6, 2.7]
+    runs-on: ${{matrix.os}}
+    services:
+      mysql:
+        image: mysql
+        ports:
+          - 3306:3306
+        env:
+          MYSQL_ROOT_PASSWORD: root
+          MYSQL_USER: github
+          MYSQL_PASSWORD: github
+          MYSQL_DATABASE: composite_primary_keys_unittest
+        options: >-
+          --health-cmd "mysqladmin ping -h localhost"
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
+    env:
+      BUNDLE_WITHOUT: "db2 oracle sqlserver sqlite postgresql"
+      BUNDLE_JOBS: 4
+      BUNDLE_PATH: vendor/bundle
+    steps:
+      - uses: actions/checkout@v2
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{matrix.ruby}}
+          bundler-cache: true
+      - name: Setup database config
+        run: cp test/connections/databases.ci.yml test/connections/databases.yml
+      - name: Prepare databases
+        run: bundle exec rake mysql:rebuild_database
```

Comment:
It uses the environment variable `MYSQL_DATABASE` to authorize the db user who connects to the mysql service, but uses `mysql:rebuild_database` instead of `mysql:build_database` to also create a database at the same time.